ccm_web/.sequelizerc

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
```

Comment:
Typescript disallow all the file other than `js`, `jsx`, `ts`, `tsx`. So in order to run migration in production I am opting to use `umzug`. 

Typescript compiler disallow even shell file and typesript has an ability to behave in shell mode. https://nathanfriend.io/2018/06/17/shelling-with-typescript.html